app/views/DependencyExplorerPage.scala.html

```diff
@@ -140,16 +140,20 @@ <h1 id="search-service-header">Dependency Explorer</h1>
                 <table id="search-results" class="table table-striped">
                     <thead>
                         <tr>
-                            <th rowspan="2"><button class="sort no-border" data-sort="service">Service</button></th>
-                            <th rowspan="2">Version</th>
-                            <th rowspan="2"><button class="sort no-border" data-sort="teams">Team</button></th>
-                            <th colspan="4" class="text-center">Dependency</th>
-                        </tr>
-                        <tr>
-                            <th>Group ID</th>
-                            <th>Artefact ID</th>
-                            <th><button class="sort no-border" data-sort="dep-version">Version</button></th>
-                            <th>Scope</th>
+                            <th rowspan="2" class="wrw-table-head"><button class="sort no-border" data-sort="service">Service</button></th>
```

Comment:
removes two columns and adds the contents to the header title
[sticky column dependency explorer.webm](https://github.com/hmrc/catalogue-frontend/assets/1241373/7d03ac1a-1df8-4b09-bb63-bdbb4fc865af)